packages/core/src/App/Components/Routes/binary-routes.jsx

```diff
@@ -16,7 +18,7 @@ const BinaryRoutes = props => {
         <React.Suspense fallback={<Loading />}>
             <Prompt when={props.prompt_when} message={props.promptFn} />
             <Switch>
-                {getRoutesConfig().map((route, idx) => (
+                {getRoutesConfig({ hide_crypto_routes: is_deriv_crypto }).map((route, idx) => (
```

Comment:
shouldn't this be `hide_non_crypto_routes`? 🤔

Reply:
true, i'll change it to `is_deriv_crypto` as seems to be the most understandable option